packages/shared/src/components/fields/MarkdownInput/index.tsx

```diff
@@ -192,7 +192,7 @@ function MarkdownInput(
             tabListProps={{ className: { indicator: '!w-6' } }}
           >
             <Tab label={CommentTab.Write}>{children}</Tab>
-            <Tab label={CommentTab.Preview} className="p-4">
+            <Tab label={CommentTab.Preview} className="min-h-[11.125rem] p-4">
```

Comment:
The actual box already has its natural `min-h` value so this won't break any existing implementation.